### PR TITLE
ServiceBus: Added MigrationState read only property to MigrationConfigurations

### PR DESCRIPTION
**Pull request by v-Ajnava**
## Description
Added MigrationState read-only property to MigrationConfigurations which reflects the state of migration
Swagger PR : https://github.com/Azure/azure-rest-api-specs/pull/4008

---

This checklist is used to make sure that common guidelines for a pull request are followed.
- [X] Please add REST spec PR link to the SDK PR
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md).**
- [X] **The pull request does not introduce [breaking changes](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/Documentation/breaking-changes.md).**

### [General Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#testing-guidelines)
- [ ] Pull request includes test coverage for the included changes.

### [SDK Generation Guidelines](https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/.github/CONTRIBUTING.md#sdk-generation-guidelines)
- [X] If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.
- [X] The generate.cmd file for the SDK has been updated with the version of AutoRest, as well as the commitid of your swagger spec or link to the swagger spec, used to generate the code.
- [X] The `*.csproj` and `AssemblyInfo.cs` files have been updated with the new version of the SDK.

### CODE REVIEW

**Comment by dsgouda** [comment]
Please update the PR with latest remote changes and squash the commits

File: src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [assembly: AssemblyDescription("Provides Microsoft Azure ServiceBus management functions for managing the Microsoft Azure ServiceBus service.")]
 
 [assembly: AssemblyVersion("1.0.0.0")]
```
Comment (dsgouda) [inline src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs]: For major version bumps, `AssemblyVersion` must also be updated

File: src/SDKs/ServiceBus/Management.ServiceBus/Generated/SdkInfo_ServiceBusManagementClient.cs [hunk 2]
```diff
@@ -18,19 +19,19 @@ internal static partial class SdkInfo
...
+                new Tuple<string, string, string>("ServiceBus", "PremiumMessagingRegions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Queues", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Regions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Rules", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Subscriptions", "2017-04-01"),
+                new Tuple<string, string, string>("ServiceBus", "Topics", "2017-04-01"),
               }.AsEnumerable();
           }
       }
-      // BEGIN: Code Generation Metadata Section
```
Comment (dsgouda) [inline src/SDKs/ServiceBus/Management.ServiceBus/Generated/SdkInfo_ServiceBusManagementClient.cs]: Please pull down latest changes from remote and run `msbuild build.proj` and regenerate the code

**Review by dsgouda — CHANGES_REQUESTED** [review]

File: src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (v-Ajnava) [reply src/SDKs/ServiceBus/Management.ServiceBus/Properties/AssemblyInfo.cs]: resolved

File: src/SDKs/ServiceBus/Management.ServiceBus/Generated/SdkInfo_ServiceBusManagementClient.cs (hunk 2 above)
Comment (v-Ajnava) [reply src/SDKs/ServiceBus/Management.ServiceBus/Generated/SdkInfo_ServiceBusManagementClient.cs]: resolved

**Review by dsgouda — CHANGES_REQUESTED** [review]
Version number and packagereleasenotes must be updated in the csproj as well

**Review by dsgouda — APPROVED** [review]
LGTM

**Pull request merged**